sdk/sqlmanagement/Azure.ResourceManager.Sql/tests/Azure.ResourceManager.Sql.Tests.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TestHelperProjects>Resources201705;Network202004;Storage201906;</TestHelperProjects>
```

Comment:
What are TestHelperProjects for?

Reply:
It is to bring in the dependency packages for tests. This is temporary and will be changed shortly using `packagereference`.